student.rb

```diff
@@ -10,6 +10,11 @@ def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def class(room)
+    @classroom = classroom
+    room.students_list.push(self) unless room.students.include?(self)
+  end
+
   def play_hooky
     "¯\(ツ)/¯"
   end
```

Comment:
- **Note: Please make visible the missing files for this repository (set up association) for a better review.** 
# in person.rb file:
![Capture](https://user-images.githubusercontent.com/63972670/194376780-5628073f-eaef-4b52-8266-b4edd89b03cf.JPG)
- You forgot to Implement " **A person has many rentals**". Please work on it

Reply:
:heavy_check_mark:  rentals method add